extra_requirements/svm.json

```diff
@@ -1,3 +1,3 @@
 {
-  "svm": ["pandas>=0.22.2,<0.24.2","openml==0.10.2","scikit-learn>=0.18.1"]
```

Comment:
We encountered a problem when having openml==0.10.2 and Pandas >24.2. Before we can allow those versions (or a newer openml version, which currently is required for the openml tasks), we need to fix this.

Reply:
see issue #51